### PR TITLE
*: Set the batch delete/insert size by session variable

### PR DESCRIPTION
**Pull request by shenli**
We hardcode it to 20k. But if the row size is large. It will still break the transaction size limitation. So we need to provide a way to set it dynamically.

### CODE REVIEW

File: sessionctx/variable/tidb_vars.go [hunk 1]
```diff
@@ -96,6 +96,10 @@ const (
 	// split data into multiple batches and use a single txn for each batch. This will be helpful when deleting large data.
 	TiDBBatchDelete = "tidb_batch_delete"
 
+	// tidb_dml_batch_size is used to split the insret/delete data into small batches. It only takes effort when tidb_batch_insert/tidb_batch_delete is on.
+	// Its default value is 20000. When the row size is large. User could change it to a smaller one to avoid break the transaction size limitation.
+	TiDBDMLBatchSize = "tidb_dml_batch_size"
```
Comment (zz-jason) [inline sessionctx/variable/tidb_vars.go]: `tidb_dml_batch_size` may be confused with `tidb_max_chunk_size`, how about `tidb_write_batch_size` ?

Comment (shenli) [reply sessionctx/variable/tidb_vars.go]: tidb_dml_batch_size is used for DML. tidb_max_chunk_size is used for select. DML includes update/insert/delete. So I think `tidb_dml_batch_size` is a proper name.

**Comment by zz-jason** [comment]
LGTM

**Comment by shenli** [comment]
@tiancaiamao @coocood @winkyao PTAL

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**